index.html

```diff
@@ -84,7 +84,41 @@ <h4>Multi-Post Stories</h4>
 </div>
```

Comment:
- [ ] On your readme file, please, kindly include the "getting started" section with instructions about how to run the project, as in the Microverse readme template.  That way, your project will be well documented 💪